### ISSUE TITLE
NPE during startup of 7.3.0

### ISSUE THREAD

**Issue by Bombe** [GoldenGnu/jeveassets#342]
Hi.

My jEveAssets just updated itself from 7.2.1 to 7.3.0 and since then refuses to startup, throwing a NullPointerException.

```
ERROR: 2022-09-18 17:48:31,502 [AWT-EventQueue-0] net.nikr.eve.jeveasset.NikrUncaughtExceptionHandler - Uncaught Exception (Thread): null
java.lang.NullPointerException: null
	at net.nikr.eve.jeveasset.data.api.my.MyIndustryJob.getMaterialEfficiency(MyIndustryJob.java:339)
	at net.nikr.eve.jeveasset.data.settings.ContractPriceManager$ContractPriceItem.create(ContractPriceManager.java:388)
	at net.nikr.eve.jeveasset.data.profile.ProfileData.updatePrice(ProfileData.java:1259)
	at net.nikr.eve.jeveasset.data.profile.ProfileData.updateEventLists(ProfileData.java:573)
	at net.nikr.eve.jeveasset.data.profile.ProfileData.updateEventLists(ProfileData.java:397)
	at net.nikr.eve.jeveasset.Program.updateEventLists(Program.java:598)
	at net.nikr.eve.jeveasset.Program.updateEventLists(Program.java:576)
	at net.nikr.eve.jeveasset.Program.<init>(Program.java:295)
	at net.nikr.eve.jeveasset.Main$1.run(Main.java:181)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:758)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:74)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:728)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:205)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

Platform is macOS 11.6.8 (M1), Java is Zulu 1.8.0_322 (8.60.0.21-CA-macos-aarch64).

**Comment by GoldenGnu** [comment]
Thanks - I'm going to look into right away

**Comment by GoldenGnu** [comment]
Fix released in 7.3.1
